productivity-suite/app/legacy-app/src/worker/script.ts

```diff
@@ -173,4 +173,60 @@ gateway.registerFragment({
   },
 });
 
+gateway.registerFragment({
+  fragmentId: "news",
+  // Note: deployment part of the url is fine also for local development since then
+  //       only the path part of the url is being used
+  getBaseUrl: () =>
+    `https://productivity-suite-news-fragment.devdash.workers.dev`,
+  prePiercingStyles: `
+    :not(piercing-fragment-outlet) > piercing-fragment-host[fragment-id="news"] {
+      position: absolute;
+      margin: 0 1rem;
+      top: 13.9rem;
+      left: 0;
+      right: 0;
+    }
+
+    @media (max-width: 45rem) {
+      :not(piercing-fragment-outlet) > piercing-fragment-host[fragment-id="news"] {
+        top: 14.05rem;
+      }
+    }
+
+    @media (max-width: 35rem) {
+      :not(piercing-fragment-outlet) > piercing-fragment-host[fragment-id="news"] {
+        top: 29rem;
+      }
+    }
+
+    @media (max-width: 25rem) {
+      :not(piercing-fragment-outlet) > piercing-fragment-host[fragment-id="news"] {
+        top: 29.65rem;
+        margin: 0;
+      }
+    }
+    `,
+  shouldBeIncluded: async (request: Request) =>
+    isPiercingEnabled(request) &&
+    (await isUserAuthenticated(request)) &&
+    /^\/(news(\/[^/]+)?)?$/.test(new URL(request.url).pathname),
```

Comment:
This also matches `/` which shouldn't as that just redirects to the todos page
![Screenshot 2022-11-02 at 09 41 52](https://user-images.githubusercontent.com/61631103/199458654-4e35eec4-0eb3-4e6f-9495-b1dd38fc7004.png)

Reply:
```suggestion
    /^\/news(\/[^/]+)?$/.test(new URL(request.url).pathname),
```